TypeCobol.Test/Parser/Programs/Cobol85/DebugNormalLines.rdzPGM.txt

```diff
@@ -0,0 +1,35 @@
+--- Diagnostics ---
+Line 14[12,20] <27, Error, Syntax> - Syntax error : In debugging mode, a statement cannot be across lines marked with debug and lines not marked debug.
+Line 19[12,19] <27, Error, Syntax> - Syntax error : In debugging mode, a statement cannot be across lines marked with debug and lines not marked debug.
+Line 24[12,19] <27, Error, Syntax> - Syntax error : In debugging mode, a statement cannot be across lines marked with debug and lines not marked debug.
+Line 29[12,19] <27, Error, Syntax> - Syntax error : In debugging mode, a statement cannot be across lines marked with debug and lines not marked debug.
+Line 35[12,18] <27, Error, Syntax> - Syntax error : In debugging mode, a statement cannot be across lines marked with debug and lines not marked debug.
+Line 40[12,17] <27, Error, Syntax> - Syntax error : In debugging mode, a statement cannot be across lines marked with debug and lines not marked debug.
+
```

Comment:
No error reported on line 58 in stacked program Pgm2 ?

Reply:
Program Pgm2 is not in debugging mode